docs/api.md

```diff
@@ -754,7 +754,7 @@ If a signal terminated the subprocess, this property is defined and included in
 
 ## Options
 
-_TypeScript:_ [`Options`](typescript.md) or [`SyncOptions`](typescript.md\
+_TypeScript:_ [`Options`](typescript.md) or [`SyncOptions`](typescript.md)
```

Comment:
```suggestion
_TypeScript:_ [`Options`](typescript.md) or [`SyncOptions`](typescript.md)\
```

Otherwise, `Type` is on the same line, instead of a different one.